CHANGELOG.md

```diff
@@ -29,6 +29,12 @@
     most stores, which don't combine multiple orders into a carton, this will
     behave the same.
 
+*   Some HABTM associations have been removed from Solidus. Specifically:
```

Comment:
I want to add a note about referential integrity here too. But seeing that Prototype <=> Taxon and Product <=> PromotionRule doesn't have it, then it might not be appropriate just yet.